src/main/scala/nl/knaw/dans/api/sword2/DepositHandler.scala

```diff
@@ -81,28 +80,26 @@ object DepositHandler {
 
   def finalizeDeposit(mimeType: String)(implicit settings: Settings, id: String): Try[Unit] = {
     log.info(s"[$id] Finalizing deposit")
-    // TODO pass on the combination object of baseDir and baseURL
-    implicit val baseDir: File = new File(settings.bagStoreBaseDir)
-    implicit val baseUrl: URI  = new URI(settings.bagStoreBaseUri)
+    implicit val optBagStoreSettings: Option[BagStoreSettings] = settings.bagStoreSettings
```

Comment:
Is this implicit picked up by `checkBagVirtualValidity` even though the parameter in that function has a different name?